vision/roadmap.md

```diff
@@ -101,7 +99,29 @@ Apps will be able to leverage an example server to subscribe users for push noti
 
 ## General Availability ✅
 
-Ready for production use. Check out [docs](https://xmtp.org/docs/client-sdk/javascript/concepts/intro-to-sdk), [tutorials](https://xmtp.org/docs/client-sdk/javascript/tutorials/build-an-xmtp-hello-world-app), and the [quickstart example](https://xmtp.org/docs/client-sdk/javascript/tutorials/quickstart).
+### Support for JavaScript and React apps
+
+Developers are able to build apps in JavaScript and React with the `xmtp-js` SDK:
+
+- [GitHub repo](https://github.com/xmtp/xmtp-js)
+- [Quickstart](/docs/client-sdk/javascript/tutorials/quickstart)
+- [Reference](/docs/client-sdk/javascript/reference/classes/Client)
+
+### Support for Swift mobile apps
```

Comment:
Is the switch in usage of Swift/iOS, Dart/Flutter, Android/Kotlin intentional? 

- If yes, where is it appropriate to use which terminology?
- If no, should we standardize across docs & repos?